xbmc/pvr/recordings/PVRRecordings.cpp

```diff
@@ -125,7 +125,7 @@ void CPVRRecordings::GetSubDirectories(const CPVRRecordingsPath &recParentPath,
       pFileItem->m_dateTime = current->RecordingTimeAsLocalTime();
 
       // Assume all folders are watched, we'll change the overlay later
-      pFileItem->SetOverlayImage(CGUIListItem::ICON_OVERLAY_WATCHED, false);
+      pFileItem->SetOverlayImage(CGUIListItem::ICON_OVERLAY_WATCHED);
```

Comment:
It's not that I insist, I just tried to be more consistent with other code.
Ideally there should be two overlay functions:
SetOverlayImage(ICON_OVERLAY) and SetOverlayImageFromPlaycount(int playcount) so that the 2nd parameter isn't needed. But this is outside of this PR ofc.